src/index.tsx

```diff
@@ -30,10 +30,10 @@ connect({
         name: 'Translate',
         type: 'addon',
         fieldTypes: [
-          Fields.textField,
-          Fields.stringField,
-          Fields.structuredTextField,
-          Fields.seo,
```

Comment:
I've renamed that to DatoFieldType as per standard it should be singular and since some part of the code import Field type from "datocms" library, I wanted to keep this separated